test/fixtures/todos.yml

```diff
@@ -0,0 +1,9 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```

Comment:
We use [FactoryGirl](https://github.com/thoughtbot/factory_girl) on our projects, so please switch over from fixtures. :-)